prow/plugins/trigger/trigger.go

```diff
@@ -275,3 +274,14 @@ func getPresubmits(log *logrus.Entry, gc git.ClientFactory, cfg *config.Config,
 	}
 	return presubmits
 }
+
+func getPostsubmits(log *logrus.Entry, gc git.ClientFactory, cfg *config.Config, orgRepo string, baseSHAGetter config.RefGetter) []config.Postsubmit {
+	postsubmits, err := cfg.GetPostsubmits(gc, orgRepo, baseSHAGetter)
+	if err != nil {
+		// Fall back to static postsubmits to avoid deadlocking when a postsubmit is used to verify
+		// inrepoconfig. Tide will still respect errors here and not merge.
```

Comment:
The description is wrong, the issue described here is not something we have for postsubmits. I guess falling back to `PostubmitsStatic` is probably still a nicer way of degrading, but we should log the error here on `Error` level, because errors here indicate a lacking validation for the `prow.yaml` or some other config issue that prevented fetching the repo